docs/hosting/configuration.md

```diff
@@ -117,11 +116,11 @@ The following environment variables support file input:
 
 ### Base URL
 
-!!! warning "Keep in mind"
-    This variable only gets used when the `n8n-editor-ui` package gets built manually. Hence, it does not get used in combination with the default n8n docker image. By default, `/` gets used, meaning that it uses the root-domain.
+!!! warning "Requires manual UI build"
+    This variable requires a manual build of the `n8n-editor-ui` package. You can't use it with the default n8n docker image. The default it`/`, meaning that it uses the root-domain.
 
 
-Tells the frontend how to reach the REST API of the backend.
+Tells the frontend how to reach the REST API of the back-end.
```

Comment:
```suggestion
Tells the front end how to reach the REST API of the back end.
```
Edited this based on the MS guide.

Reply:
Heh my hyphen was a compromise - I prefer backend, MS doesn't like it. But you're right, we should stick to the guide we've chosen.